sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptionVersion.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.specialized.cryptography;
+
+/**
+ * Specifies the version of the client side encryption protocol to use.
+ * V1 uses AES/CBC which is no longer secure. It is supported for compatibility but should not be used for new uploads.
```

Comment:
I'd push this Javadoc into the V1 Javadoc. I'd also give a small explanation on each version about what encryption algorithms are being used, this could help situations where the underlying JVM doesn't have support for the algorithm